provision-osx/README.md

```diff
@@ -12,144 +12,145 @@ Provision Jenkins node for the build farm. This role performs a few tasks, these
 
-### Options
+### Variables
 * `homebrew_version` - The version of Homebrew to install (git tag).
 * `homebrew_packages` - The packages to install using Homebrew. Format: `{ name: <PACKAGE_NAME> }`.
 * `homebrew_repo` - The git repo where Homebrew resides (defaults to GitHub repo).
-* `homebrew_prefix` -
+* `homebrew_prefix` - (???)
 * `homebrew_install_path` - Where Homebrew will be installed, used as `homebrew_prefix/homebrew_install_path`.
-* `homebrew_brew_bin_path` - Where `brew` will be installed.
+* `homebrew_brew_bin_path` - `brew` will be installed in this directory.
 * `homebrew_taps` - A list of taps to add.
 
-## Install RVM and Ruby
-Installs RVM along with a single version of Ruby. The version of Ruby
-to install can be defined by a variable. The cocoapods gem will be installed,
-other gems can be specified using the `gem_packages` variable.
+## Install RVM, Ruby and Cocoapods
+To run this section as a standalone step, use `osx_install_ruby` tag in the playbook.
 
-To run this section as a standalone step you must specify the `osx_install_ruby` tag.
+Installs RVM, a single version of Ruby and `cocoapods` gem. Other gems can be installed by configuring the `gem_packages` variable.
 
-### Options
+
+### Variables
 * `ruby_version` - The version of Ruby to install on the node.
-* `cocoapods_version` - The version of the Cocoapods gem to install.
+* `cocoapods_version` - The version of the `cocoapods` gem to install.
 * `gem_packages` - A list of gems to install on the node. Format: `{ name: <PACKAGE_NAME>, version: <PACKAGE_VERSION> }`.
 
 ## Install NVM and Node
-Installs NVM along with any versions of Node specified through variables.
-Packages from NPM can be defined also, however they will be installed globally.
+To run this section as a standalone step, use `osx_install_nodejs` tag in the playbook.
 
-To run this section as a standalone step you must specify the `osx_install_nodejs` tag.
+Installs NVM along with versions of Node specified in variables.
+Packages from NPM can be specified for installation too, but they will be installed globally.
 
-### Options
+
+### Variables
 * `nvm_install_url` - The URL of NVM installation script (defaults to GitHub release).
-* `nvm_install_file_name` - What to name the file on the node.
+* `nvm_install_file_name` - What to name the file on the node. (???)
 * `node_versions` - A list of Node versions to install.
 * `npm_packages` - A list of global NPM packages to install. Format: `{ name: <PACKAGE_NAME>, version: <PACKAGE_VERSION> }`.
 
-## Install Xcode
-Installs Xcode CLI tools and Xcode versions specified.
-Installing Xcode can take some time (~ 30 minutes without a cached copy) so please
-be patient during these stages.
+## Install Xcode and Xcode CLI tools
+To run this section as a standalone step use `osx_install_xcode` tag in the playbook.
+
+***Installing Xcode can take ~ 30 minutes without a cached copy.***
```

Comment:
btw: It's 30 minutes **per installed version**, maybe we should add this piece of information while we are at it..

Reply:
I have experienced random timeouts from apple server as well

Reply:
done